modules/dailyhuntBidAdapter.js

```diff
@@ -59,134 +341,55 @@ export const spec = {
 
   supportedMediaTypes: SUPPORTED_MEDIA_TYPES,
 
-  isBidRequestValid: bid => !!bid.params.partnerId,
+  isBidRequestValid: bid => !!bid.params.placement_id && !!bid.params.publisher_id && !!bid.params.partner_name,
```

Comment:
not sure if you also want to check validity of things like mediatype here, but it's not blocking

Reply:
Hello @bmwcmw ,

Greetings from dailyhunt.
Thanks for reviewing. It is not required now.